clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -138,7 +133,13 @@ class AzureResourceManagerClient extends AzureBaseClient {
     ResourceManagementService.create(this.buildConfiguration(creds))
   }
 
-  static DeploymentExtended createTemplateDeploymentFromPath(
+  private static void createResourceGroupVNet(AzureCredentials creds, String resourceGroupName, String region) {
+    def vNetName = AzureUtilities.VNET_NAME_PREFIX + resourceGroupName
+
+    creds.getNetworkClient().createVirtualNetwork(creds, resourceGroupName, vNetName, resourceGroupName, "10.0.0.0/16")
```

Comment:
you're passing the resourceGroupName for the region (which is what is causing the createLoadBalancer operation to fail). Also you do not need to address in this call as the method will default to that value.